Disable fallback to smaxage for our symfony  http cache customttllistener handling (#7540)
sulu/sulu: UPGRADE.md

The image formats URL requires an exact filename match to retrieve the correct image format. 
Old versions will be redirected to the new version and any non-matching filenames will now return a 404 error.

> ### The s-maxage header does not longer effect server side http caching

Reviewer: ```suggestion
### The s-maxage header no longer affects server side http caching
```

> Sulu accidently cached big media files of the image / thumbnail generator controller.
> As it was never expected that Sulu caches based on `s-maxage` behaviour instead it always did use
> its own Custom TTL Header `X-Reverse-Proxy-TTL` instead.
> The CustomTtlListener currently did unexpectly fallback to `s-maxage`

Reviewer: ```suggestion
Sulu accidentally cached large media files generated by the image/thumbnail generator controller. It was never expected that Sulu would cache based on the `s-maxage` behavior. Instead, it was supposed to use its own custom TTL header `X-Reverse-Proxy-TTL`. However, the CustomTtlListener unexpectedly fell back to the `s-maxage` header.
```

> An upgrade of the `friendsofsymfony/http-cache` to atleast `2.16.0` or `3.1.0` is required.

Reviewer: ```suggestion
An upgrade of `friendsofsymfony/http-cache` to at least `2.16.0` or `3.1.0` is required.
```

```bash
composer update friendsofsymfony/http-cache
```

> For custom controllers which you currently did cache on `s-maxage` define also the `X-Reverse-Proxy-TTL`
> if you still want to cache that response in the Symfony Http Cache:

Reviewer: ```suggestion
For custom controllers that currently use `s-maxage` for caching, also define the `X-Reverse-Proxy-TTL` header
if you still want to cache that response in the Symfony Http Cache:
```
